### PR TITLE
[6.x] package docker container images with mage

### PR DESCRIPTION
**Pull request by graphaelli**
backports #1543 to 6.x + beats framework update (required for docker packaging)

### CODE REVIEW

**Comment by graphaelli** [comment]
sync job will be fixed by #1633

File: _beats/dev-tools/packaging/packages.yml [hunk 1]
```diff
@@ -157,6 +157,22 @@ shared:
...
+      linux_capabilities: ''
+    files:
+      '{{.BeatName}}.yml':
+        source: '{{.BeatName}}.docker.yml'
+        mode: 0600
+        config: true
+
+  - &elastic_docker_spec
+    extra_vars:
+      repository: 'docker.elastic.co/beats'
```
Comment (jalvz) [inline _beats/dev-tools/packaging/packages.yml]: does this play well with us?

**Comment by jalvz** [comment]
i think this is packaging 6.6.0 and 7.0.0, is that intended?

File: _beats/dev-tools/packaging/packages.yml (hunk 1 above)
Comment (graphaelli) [reply _beats/dev-tools/packaging/packages.yml]: yeah, we override this here: https://github.com/elastic/apm-server/pull/1632/files#diff-3db01cb68c42f27f6422c65fa1175e23R193

**Pull request merged**